spec/models/user_spec.rb

```diff
@@ -38,4 +39,22 @@
       expect(user_artist.fan?).to eq false
     end
   end
+
+  describe 'Facebook OAuth methods' do
+    let(:auth_response) { OmniAuth::AuthHash.new(OmniAuthFixtures.facebook_mock) }
+
+    it 'creates an instance from an oauth hash' do
+      create_user = lambda {User.from_omniauth(auth_response)}
+      expect{create_user.call}.to change{ User.count}.from(0).to(1)
+    end
+  end
```

Comment:
Nice spec. I like it.

Reply:
You could also test if the `provider` is set to `facebook` that would be even better.